Platforms/QemuSbsaPkg/PlatformBuild.py

```diff
@@ -391,14 +391,14 @@ def PlatformPostBuild(self):
                 },
                 "example": {
                     "image": {
-                        "file": os.path.join(self.env.GetValue('BUILD_OUTPUT_BASE'), 'FV', 'BL32_AP_MM_SP1.fd'),
+                        "file": "/home/rdiaz/Projects/trusted-services/deployments/ms-services/config/default-sp/build/b8bcbd0c-8e8f-4ebe-99eb-3cbbdd0cd412.bin",

```

Comment:
You'll need to get rid of the user-local path before merging.

Reply:
No worries, I am aware that this needs to change. There are plans to generate the SP using UEFI instead of Trusted Services which is where this binary comes from.

Reply:
Sounds good. It is okay to have a temporary path, but it should be relative to the workspace or if it is truly an example where the explicitness of an absolute path helps, anonymize it.